roles/matrix-nginx-proxy/templates/nginx/conf.d/matrix-jitsi.conf.j2

```diff
@@ -53,20 +53,25 @@
 
 		tcp_nodelay on;
 	}
-
+	{% if matrix_nginx_proxy_proxy_jitsi_additional_jvbs_enabled %}
+	{% for jvb in matrix_nginx_proxy_proxy_jitsi_additional_jvbs %}
```

Comment:
There should be a way to iterate over this without 2 nested `for` loops.

With my other review suggestion (which redoes `matrix_nginx_proxy_proxy_jitsi_additional_jvbs` to be a dictionary, instead of a list of a dictionaries), the need for 2 `for` loops is removed.

You can use `{% for id, ip_address in matrix_nginx_proxy_proxy_jitsi_additional_jvbs %}` here.

You also don't need the `matrix_nginx_proxy_proxy_jitsi_additional_jvbs_enabled` part.

Reply:
I added in the enabled flag but after testing with it I agree its not enabled. I can remove the double loop and use a dictionary instead. 👍